uefi/src/main.rs

```diff
@@ -63,39 +69,77 @@ fn efi_main(image: Handle, st: SystemTable<Boot>) -> Status {
 
 fn main_inner(image: Handle, mut st: SystemTable<Boot>) -> Status {
     // temporarily clone the y table for printing panics
+
     unsafe {
         *SYSTEM_TABLE.get() = Some(st.unsafe_clone());
     }
+    let stdout = unsafe { &mut *SYSTEM_TABLE.get() };
+    let stdout = stdout.as_mut().unwrap();
+    let stdout = stdout.stdout();
+    stdout.clear().unwrap();
```

Comment:
We don't want to hold on to the `&mut` stdout reference in case a panic happens. So instead of defining a `stdout` binding, we want to use `st.stdout()` directly. This also applies to the `writeln!` calls below.

```suggestion
    st.stdout().clear().unwrap();
```

Reply:
This was originally done during test debugging where it appeared boot was hanging (But was a TFTP boot failure).

Without grabbing the SYSTEM_TABLE pointer, and getting stdout from there, I couldn't figure out how to get it compile :sweat_smile: 

Fixed.